src/app/rundown-view/components/rundown-tooltips/mini-shelf-tooltip/mini-shelf-tooltip.component.html

```diff
@@ -0,0 +1,6 @@
+<div #tooltipHoverElement (mouseenter)="emitNewHoverMouseEvent($event)" (mousemove)="emitNewHoverMouseEvent($event)" (mouseleave)="emitNewHoverMouseEvent()" class="c-hover-scrub-hover-element"></div>
```

Comment:
You can utilize Angulars `HostListener` and just bind the event directly in the typescript:
![image](https://github.com/tv2/ng-sofie/assets/23049647/84b37ae9-de24-4743-bbc0-c730a3fbaa40)

Reply:
@LindvedKrvang We can, but I don't see any benefit of doing that, instead emit event on to the element, or I miss something?

Reply:
Done